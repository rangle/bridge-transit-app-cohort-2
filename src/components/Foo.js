import React, {Component} from 'react';

class Foo extends Component {
  render() {
    return <div>Hello World</div>;
  }
}

export default Foo;
