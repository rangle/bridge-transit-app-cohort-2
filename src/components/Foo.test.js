import React from 'react'
import { shallow } from 'enzyme'
import Foo from './Foo';

describe('Foo', () => {
  it('renders correctly', () => {
    const component = shallow(<Foo />);
    expect(component).toMatchSnapshot();
  })
});