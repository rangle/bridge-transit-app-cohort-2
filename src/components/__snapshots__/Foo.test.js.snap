// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Foo renders correctly 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div>
    Hello World
</div>,
  "nodes": Array [
    <div>
      Hello World
</div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <Foo />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": Foo {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div>
          Hello World
</div>,
        "_debugID": 2,
        "_renderedOutput": <div>
          Hello World
</div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Foo />,
}
`;
